### ISSUE TITLE
Problem with Waffle not automatically remove the labels

### ISSUE THREAD

**Issue by johlju** [PowerShell/DscResources#475]
As mentioned here in the Automation.md under the [Waffle section](https://github.com/PowerShell/DscResources/blob/master/Automation.md#waffle) (some) labels should be removed when a PR is closed or merged, or when an Issue is closed. This seems to have stopped working.

One thought is that this might be a result of me adding/saving all the repositories when I previously was a member of the PowerShell GitHub organization, thus having more permission. Since I no longer have that permission the problem can be as simple as @kwirkykat make a change so the permission is updated. 

Of what I remember there should be no need for the Waffle page to be open for this to work, but not really sure about that either now, because I know there is a bot that can be installed, but since everything worked without it I never looked into it.

The whole point of the Waffle board removing labels was so that a few clicks of work could be removed from the maintainers. 🙂

**Comment by gaelcolas** [comment]
Hey @johlju, I might need to pick your brain on that one. ;)

**Comment by johlju** [comment]
**Background**
No problem 😃 When ever an issue or PR was closed Waffle would remove the labels on the PR or Issue it that label was connected to a column in the Waffle board. It essentials automatically remove 'needs review' etc. 
All labels should not (or at least it was my thought at that time) be connected to a column since we want to keep some labels, like bug, enhancement and tests etc.
The first "half" of the board is issue columns (labels), and the second "half" is PR related columns.

The most important labels are 'help wanted','in progress', 'needs review', and the 'waiting...' labels. 
The 'help wanted' label moves the issue away from the backlog and to the 'Help Wanted' column (since that label is connected to that column).
The 'needs review' or 'waiting...' labels are moving the PR away from the backlog to the respective column.

**Issue**
Waffle was always removing or adding labels (if move on the Waffle board) automatically. I did not see the need to install the Waffle Bot. The problem I think (I'm guessing here) happened is that when I added new repositories to the Waffle board, my account (which then had permission) was used to access all the repos. When my permission was removed my account which Waffle used no longer had permission.
So the solution might be either

- Katie uses her account to update the repositories in Waffle (maybe just add DscResource.Template) so they use her permissions (her account probably won't loose permission anytime soon).
- The Waffle Bot is installed on all the repos. I didn't read up on this since I didn't see the need for it, so not sure what needs to be done to get it to work.
